packages/block-library/src/latest-posts/edit.js

```diff
@@ -479,15 +480,22 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 								.trim()
 								.split( ' ', excerptLength )
 								.join( ' ' ) }
-							{ /* translators: excerpt truncation character, default …  */ }
-							{ __( ' … ' ) }
-							<a
-								href={ post.link }
-								rel="noopener noreferrer"
-								onClick={ showRedirectionPreventedNotice }
-							>
-								{ __( 'Read more' ) }
-							</a>
+							{ createInterpolateElement(
+								/* translators: excerpt truncation character, default …  */
+								__( ' … <a>Read more</a>' ),
```

Comment:
I added this for i18n. Locales should have be able to determine where the ellipsis is, if at all.

It's not the subject of the bug report however and can be rolled back, and the backend changed to match.